src/styles/card.css

```diff
@@ -0,0 +1,105 @@
+/*  CARD BLOCK:  */
+
+.card {
+  padding: 32px 0 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
```

Comment:
Better to set `box-sizing: border-box`, then calculate needed width to fit borders